packages/perspective-viewer/src/js/viewer.js

```diff
@@ -484,7 +484,7 @@ class PerspectiveViewer extends ActionElement {
             }
         }
         if (this._plugin.save) {
-            obj.plugin_config = this._plugin.save();
+            obj.plugin_config = this._plugin.save.call(this);

```

Comment:
It looks like it should already do this, based on the way the other methods are could.

Maybe it's worth submitting this file's change directly to Perspective as a separate PR?

Reply:
I'll take a look at this now.